Doppler.Sap.Test/SapTaskHandlerTest.cs

```diff
@@ -42,17 +42,16 @@ public async Task SapTaskHandler_ShouldBeUseSameCookies_WhenTimeSessionIsMinorTh
             sapConfigMock.Setup(x => x.Value)
                 .Returns(new SapConfig
                 {
-                    BaseServerUrl = "http://123.123.123",
-                    CompanyDB = "CompanyDb",
-                    Password = "password",
-                    UserName = "Name"
+                    //BaseServerUrl = "http://123.123.123"
                 });
 
             var sapTaskHandler = new SapTaskHandler(
-                sapConfigMock.Object,
+                new SapConfig { },
```

Comment:
If you don't use `sapConfigMock`, you can remove this variable in line 42

Reply:
Done